packages/flutter/test/widgets/selectable_region_test.dart

```diff
@@ -16,8 +16,8 @@ import 'semantics_tester.dart';
 
 Offset textOffsetToPosition(RenderParagraph paragraph, int offset) {
   const Rect caret = Rect.fromLTWH(0.0, 0.0, 2.0, 20.0);
-  final Offset localOffset = paragraph.getOffsetForCaret(TextPosition(offset: offset), caret);
-  return paragraph.localToGlobal(localOffset);
+  final Offset localOffset = paragraph.getOffsetForCaret(TextPosition(offset: offset), caret) + Offset(0.0, paragraph.preferredLineHeight);
+  return paragraph.localToGlobal(localOffset) + const Offset(kIsWeb ? 1.0 : 0.0, -2.0);
```

Comment:
I borrowed this from https://github.com/flutter/flutter/blob/84f65686d99954972f400e82966e7eff31d4c9f2/packages/flutter/test/widgets/editable_text_utils.dart#L50 , I couldn't really track down why this change was needed but it originally came from https://github.com/flutter/flutter/pull/4397